docs/browser-acceptance-tests.md

```diff
@@ -1,178 +0,0 @@
-      </tr>
-      <tr>
-         <td>Declarative Vs Imperative Scenarios</td>
-         <td>
-            The declarative style describes behavior at a higher level, which improves the readability of the feature by abstracting out the implementation details of the application.  The imperative style is more verbose but better describes the expected behavior.  Either style is acceptable.<br><br>
-<u>Example: Declarative</u>
-<pre>
-Scenario:User logs in
-  Given I am on the homepage
-  When I log in
-  Then I should see a login notification
-</pre>
-<u>Example: Imperative</u>
-<pre>
-Scenario: User logs in
-  Given I am on the homepage
-  When I click on the "Login" button
-  And I fill in the "Email" field with "me@example.com"
-  And I fill in the "Password" field with "secret"
-  And I click on "Submit"
-  Then I should see "Welcome to the app, John Doe"
-</pre>
-         </td>
-      </tr>
-   </tbody>
-</table>
-
-#### Further reading
-
-- [Cucumber features](https://github.com/cucumber/cucumber/wiki/Feature-Introduction)
-- [Protractor](https://angular.github.io/protractor/#/)
-- [Select elements on a page](https://www.seleniumhq.org/docs/03_webdriver.jsp#locating-ui-elements-webelements)
-- [Writing Jasmin expectations](https://jasmine.github.io/2.0/introduction.html#section-Expectations).
-- [Understanding Page Objects](https://www.thoughtworks.com/insights/blog/using-page-objects-overcome-protractors-shortcomings)
-
-## Performance testing
-
-To audit if the site complies with performance best practices and guidelines,
-[Google's Lighthouse](https://github.com/GoogleChrome/lighthouse) can be run
-from Google Chrome by opening the developer console and going to the Lighthouse
-tab to run a performance audit.
-
-## Accessibility testing
-
-Run the acceptance tests with an `--a11y` flag (i.e. `gulp test:acceptance --a11y`)
-to check every webpage for WCAG and Section 508 compliancy using Protractor's
-[accessibility plugin](https://github.com/angular/protractor-accessibility-plugin).
-
-## Source code linting
-
-The default test task includes linting of the JavaScript source, build,
-and test files.
-Use the `gulp lint` command from the command-line to run the ESLint linter,
-which checks the JavaScript against the rules configured in `.eslintrc`.
-[See the ESLint docs](https://eslint.org/docs/rules/)
-for detailed rule descriptions.
-
-There are a number of options to the command:
-
- - `gulp lint:build`: Lint only the gulp build scripts.
- - `gulp lint:test`: Lint only the test scripts.
- - `gulp lint:scripts`: Lint only the project source scripts.
- - `--fix`: Add this flag (like `gulp lint --fix` or `gulp lint:build --fix`).
-   to auto-fix some errors, where ESLint has support to do so.
- - `--path`: Add this flag to specify a file to lint,
-   rather than all files. Path is relative to the project root,
-   such as `gulp lint --path=cfgov/unprocessed/js/modules/Analytics.js`.
```

Comment:
We should retain these sections that didn't really correspond to the Cucumber tests. Could you restore them and rename this file to "Other Front-end Testing" (and update the nav link accordingly in `mkdocs.yml`)?

Reply:
Done